core/build.sh

```diff
@@ -16,6 +16,14 @@ cd "$THIS_SCRIPT_PATH"
 
 ################################ Main script ################################
 
+cleanup_visual_studio_path
+
+MESON_LOW_VERSION=false
+
+if [[ `meson --version` < 0.54 ]]; then
+  MESON_LOW_VERSION=true
+fi
```

Comment:
I see we're adding this check, despite requiring a minimum version of 1.0?

Reply:
We only require 1.0 for Windows. For Linux we have 0.53.x due to constraints on Debian packaging agents.